cli/lib/tasks/install.js

```diff
@@ -278,45 +202,32 @@ const start = (options = {}) => {
     logger.log()
   }
 
+  const pkgVersion = util.pkgVersion()
+  const versionToInstall = getVersionOverride({ envVarVersion }) || pkgVersion
```

Comment:
total nit, but for the sense of flow while reading, could we move this check above for readability?
```
const pkgVersion = util.pkgVersion()
const versionToInstall = getVersionOverride({ envVarVersion }) || pkgVersion

if (versionToInstall === '0') {
```